### ISSUE TITLE
Add some way to hide commands from the "Run a macro" selection box

### ISSUE THREAD

**Issue by t-gehring** [jeff-hykin/macro-commander#3]
As a macro can run other macros, it can be helpful to define "utility" macros, not intended to be run on their own but instead to perform common macro steps that would otherwise be duplicated between various macros. It would be useful to have a way to prevent these utility macros from showing in the "Run a macro" box, to reduce visual clutter, and to prevent them accidentally being run individually.

I would suggest something like a prefix on the macro name, such as "_." or "hidden." or "util.", as this is easy to add into the settings without adding an extra layer of JSON hierarchy.

**Comment by jeff-hykin** [comment]
Yeah I think an `_` at the front of the name would work well for this. I'll add a one time warning encase people do it on accident.

**Comment by jeff-hykin** [comment]
It might be nice to have a way to define JavaScript functions that are accessible to all of the macros too since there could be a lot of repeated code there.
